components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/devportal-api.yaml

```diff
@@ -1206,14 +1225,15 @@ paths:
   /applications/{applicationId}:
     get:
       tags:
-      - Applications
+        - Applications
       summary: |
         Get Details of an Application
       description: |
         This operation can be used to retrieve details of an individual application specifying the application id in the URI.
       parameters:
-      - $ref: '#/components/parameters/applicationId'
-      - $ref: '#/components/parameters/If-None-Match'
+        - $ref: '#/components/parameters/organizationId'
```

Comment:
I think we don't need organizationId here?
Can you please check and remove from all the unwanted places?

Reply:
Fixed https://github.com/wso2/carbon-apimgt/pull/9684/commits/0674510dd7d56bfe90e23dcc59886ea7b77cf741